app/string.js

```diff
@@ -0,0 +1,52 @@
+String.prototype.hasVowels = function(){
+    return /[aeiou]/i.test(this);
+  };
+String.prototype.toUpper = function() {
+    return this.replace(/[a-z]/g, (lowerChar) => {
+      return String.fromCharCode(lowerChar.charCodeAt(0)-32);
+    });
+  };
+String.prototype.toLower = function() {
+    return this.replace(/[A-Z]/g, (upperChar) => {
+      return String.fromCharCode(upperChar.charCodeAt(0)+32);
+    });
+};
+String.prototype.ucFirst = function() {
+    return this.replace(/[a-z]/, (upperFirst) => {
```

Comment:
This does not work for cases where the first character is already uppercase. For example, running `'Kevin'.ucFirst()` returns `KEvin`